### PR TITLE
Update tiny model summary file for recent models

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

I just created tiny models for recent models. This PR updates tiny model summary file to use them. This includes:
- TFBartForSequenceClassification
- TFBlipForConditionalGeneration
- ClapModel
- MegaModel
- NllbMoeModel
- TFVisionTextDualEncoderModel
- WhisperForAudioClassification

A few tiny changes are included too.

### CODE REVIEW

File: tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py [hunk 1]
```diff
@@ -127,8 +128,19 @@ def create_and_check_model(
...
+        {
+            "conversational": GPTSanJapaneseForConditionalGeneration,
+            "feature-extraction": GPTSanJapaneseForConditionalGeneration,
+            "summarization": GPTSanJapaneseForConditionalGeneration,
+            "text2text-generation": GPTSanJapaneseForConditionalGeneration,
+            "translation": GPTSanJapaneseForConditionalGeneration,
+        }
+        if is_torch_available()
+        else {}
+    )
```
Comment (ydshieh) [inline tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py:143]: add the attribute

File: tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py [hunk 2]
```diff
@@ -140,6 +152,19 @@ class GPTSanJapaneseTest(ModelTesterMixin, unittest.TestCase):
...
+        self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
+    ):
+        if pipeline_test_casse_name == "SummarizationPipelineTests":
+            # TODO: fix `_reorder_cache` is not implemented for this model
+            return True
+        elif pipeline_test_casse_name == "Text2TextGenerationPipelineTests":
+            # TODO: check this.
+            return True
+
+        return False
```
Comment (ydshieh) [inline tests/models/gptsan_japanese/test_modeling_gptsan_japanese.py:166]: Keep track the failing tests.

File: tests/utils/tiny_model_summary.json
Comment (ydshieh) [inline tests/utils/tiny_model_summary.json:1]: just information about the tiny models on the Hub

File: utils/create_dummy_models.py [hunk 3]
```diff
@@ -1329,6 +1335,33 @@ def build_simple_report(results):
...
+
+    updated_data = {}
+    for key in sorted(data.keys()):
+        updated_data[key] = {}
+        for attr, value in data[key].items():
+            # deduplication and sort
+            updated_data[key][attr] = sorted(set(value)) if attr != "sha" else value
+
+    with open(os.path.join(report_path, "updated_tiny_model_summary.json"), "w") as fp:
+        json.dump(updated_data, fp, indent=4, ensure_ascii=False)
```
Comment (ydshieh) [inline utils/create_dummy_models.py:1362]: moved from `utils/update_tiny_models.py` (just to make things easier)

**Review by sgugger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**